src/components/EditorWindow.tsx

```diff
@@ -148,23 +173,25 @@ const EditorWindow:FunctionComponent<EditorWindowProps> = ({onExecuteQuery}) =>
       title: "Open",
       iconProps: { iconName: "OpenFolderHorizontal" },
       onClick: () => {
-        ipcRenderer.send("show-open-dialog")
+        loadScript();
+      }
+    },
+    {
+      key: "save-as",
```

Comment:
Rather than having 2 similar icons and the menu getting cluttered I think it would be nicer to use the menu button functionality to essentially make it a drop down. See the second IconButton example [here](https://developer.microsoft.com/en-us/fluentui#/controls/web/button)

Reply:
I think that's a good call, but I'd prefer to leave that until we have a bit more of the functionality fleshed out then look at the UI in its entirety. 

Currently the app is a bit 'button heavy', similar to the current KDB Studio:

![image](https://user-images.githubusercontent.com/1098110/128512699-9858490e-f6f3-423c-a2ac-e1dafdada068.png)

Whereas, comparing to VS Code itself, most of those functions are relegated to menus and hotkeys. People know where to look for load / save etc ...

Reply:
See also https://github.com/ScottLogic/kdb-studio-2/issues/49